src/logics/LRALogic.cc

```diff
@@ -173,25 +173,11 @@ PTRef LRALogic::insertTerm(SymRef sym, vec<PTRef> &&terms) {
  * @param sum
  * @return Constant part of the normalized sum as LHS and non-constant part of the normalized sum as RHS
  */
-opensmt::pair<opensmt::Number, PTRef> LRALogic::sumToNormalizedPair(PTRef sum) {
-    assert(isNumPlus(sum));
-    vec<PTRef> varFactors;
-    PTRef constant = PTRef_Undef;
-    Pterm const & s = getPterm(sum);
-    for (PTRef arg : s) {
-        if (isConstant(arg)) {
-            assert(constant == PTRef_Undef);
-            constant = arg;
-        } else {
-            assert(isLinearFactor(arg));
-            varFactors.push(arg);
-        }
-    }
 
-    if (constant == PTRef_Undef) { constant = getTerm_NumZero(); }
-    opensmt::Number constantVal = getNumConst(constant);
-    assert(varFactors.size() > 0);
-    termSort(varFactors);
+opensmt::pair<FastRational, PTRef> LRALogic::sumToNormalizedPair(PTRef sum) {
```

Comment:
I'm going to change this into 

`opensmt::pair<opensmt::Number, PTRef> LRALogic::sumToNormalizedPair(PTRef sum) {`

Reply:
OK!